package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
I created a branch with the overrides so you can see that they allow the app to correctly load: https://github.com/timbur/another-systemjs-test/tree/using-overrides